src/common/Tile/DetailsTile/styled.js

```diff
@@ -0,0 +1,91 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  max-width: 1368px;
+  padding: 16px;
+  box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
```

Comment:
If you think there should be a shadow, let's leave it that way, but there is no such thing in Figma. ;)

Reply:
You're right - it's not in Figma - weird. Some tiles have a shadow and some do not. Maybe I'll leave it, it looks good to me 😜